lib/terraform/runner.rb

```diff
@@ -9,32 +9,59 @@ class << self
       def available?
         return @available if defined?(@available)
 
-        response = terraform_runner_client.get('ping')
-        @available = response.status == 200
+        response = terraform_runner_client.get('live')
+        @available = response.status == 200 && JSON.parse(response.body)['status'] == 'UP'
```

Comment:
@putmanoj change looks unrelated to the retirement work, can you pull it out into its own PR?

Reply:
This PR was created long back, yes will move to another PR